docs/content/preview/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md

```diff
@@ -0,0 +1,81 @@
+
+## Users
+
+A user can interact with a YugabyteDB Anywhere through the UI or REST API.
+
+Users are assigned roles, which determine what they actions they can perform.
+
+### Create, modify, and delete users
+
+As a Super Admin or Admin, you can invite new users and manage existing users for your YugabyteDB Anywhere instance.
+
+You can invite new users as follows:
+
+1. Navigate to **Admin > User Management > Users**, and click **Add User**.
+
+1. Complete the fields of the **Add User** dialog shown in the following illustration:
+
+    ![Add User](/images/yp/authorization-platform/add-user.png)
+
+1. Click **Submit**.
+
+To modify a user role or delete the user, navigate to **Admin > User Management > Users**. Click **Actions** that corresponds to the specific user, and then choose either **Edit User Role** or **Delete User**.
+
+## Roles
+
+YugabyteDB Anywhere includes built-in roles, and you can define custom roles for team members to restrict access to specific account features.
+
+[API tokens](../../anywhere-automation/) generated for users are assigned the same role as the user that generated them.
+
+### Built-in roles
+
+The following built-in roles are available:
+
+* **Super Admin** is the first user that is created during the installation. This role has the highest level of privilege that allows all read and write actions on all YugabyteDB Anywhere resources. There can be only one Super Admin. This Super Admin can perform the following:
+
+  * Manage all resources, including universes, nodes, backup, restore, and cloud providers.
+  * Manage the user access control by creating and managing users.
+
+  For more information, see [admin user](../../configure-yugabyte-platform/create-admin-user/).
+
+* **Admin** has privileges that are similar to the Super Admin, except that the Admin cannot manage the global scope artifacts and actions, such as runtime configuration settings and LDAP authentication.
+
+* **Backup Admin** has access to the backup related tasks, such as the following:
+
+  * Manage database backups and restore operations.
+  * Create new backups.
+  * Delete any existing backup or backup schedule.
+  * Edit existing backups.
+  * Read-only permissions for all other resources in YugabyteDB Anywhere.
+
+* **Read-only** access level provides viewer permission to the UI and API.
+
+### Create, modify, and delete roles
+
+As a Super Admin or Admin, you can create and modify roles.
+
+To create a custom role, do the following:
+
+1. Navigate to **Admin > User Management > Roles**, then click **Add Role** to display the **Create a Role** dialog.
```

Comment:
Admin > Access Management > Roles

Reply:
With the new RBAC, we can not only create Custom Roles but also scope the customer roles at the universe level.

Please also include the flow to modify the role bindings of the users from the Admin > Access Management > Users > Edit Assigned Roles option for the user.

Reply:
The new RBAC feature which allows custom roles can be turned on by using the runtime flag yb.rbac.use_new_authz. This is an internal Runtime config as of now. I will make the change to PUBLIC once I get the QA approval.

The runtime config can be edited from the UI from the Admin > Advanced > Global Configuration section and the runtime config key yb.rbac.use_new_authz should be set to true for the user to use the new RBAC logic. Once the runtime config is set to true, it cannot be turned off

Reply:
Please write a section on how to assign universes for Custom roles

Reply:
Phase 1 MVP: The resource that we deal with are Universes. Later phases will deal with other kind of resources